modules/spa-s3-cloudfront/main.tf

```diff
@@ -54,7 +54,7 @@ locals {
 # Create an ACM and explicitly set it to us-east-1 (requirement of CloudFront)
 module "acm_request_certificate" {
   source  = "cloudposse/acm-request-certificate/aws"
-  version = "0.17.0"
+  version = "0.16.3"
```

Comment:
this is intentional. The latest version is `0.16.3`
https://github.com/cloudposse/terraform-aws-acm-request-certificate/releases